src/scripting/lua_color.cpp

```diff
@@ -126,11 +126,28 @@ static int impl_color_get(lua_State *L)
 	return 0;
 }
 
+static int impl_color_dir(lua_State* L)
+{
+	static const std::vector<std::string> keys{"min", "max", "mid", "minimap", "pango_color"};
+	lua_push(L, keys);
+	return 1;
+}
+
 static int impl_color_set(lua_State *L)
 {
 	return luaL_argerror(L, 2, "color objects canot be modified");
 }
 
+static int impl_colors_dir(lua_State* L)
```

Comment:
I'd prefer to have more than a one-character difference from `impl_color_dir`. Even if it's trivial in this case, it's something that I'd like to be a general coding guideline.

Maybe `impl_colors_table_dir`?